tsc --build error

```
src/components/Drawer/Drawer.tsx:97:3 - error TS2742: The inferred type of 'render' cannot be named without a reference to 'react-dom/node_modules/@types/react'. This is likely not portable. A type annotation is necessary.

97   render() {
     ~~~~~~

src/components/Popup/Popup.tsx:144:3 - error TS2742: The inferred type of 'createPopElement' cannot be named without a reference to 'react-dom/node_modules/@types/react'. This is likely not portable. A type annotation is necessary.

144   createPopElement = () => {
      ~~~~~~~~~~~~~~~~

src/components/Popup/Popup.tsx:223:3 - error TS2742: The inferred type of 'render' cannot be named without a reference to 'react-dom/node_modules/@types/react'. This is likely not portable. A type annotation is necessary.

223   render () {
      ~~~~~~

src/components/Portal/Portal.tsx:97:3 - error TS2742: The inferred type of 'render' cannot be named without a reference to 'react-dom/node_modules/@types/react'. This is likely not portable. A type annotation is necessary.

97   render() {
     ~~~~~~

src/components/PurePortal/PurePortal.tsx:70:3 - error TS2742: The inferred type of 'render' cannot be named without a reference to 'react-dom/node_modules/@types/react'. This is likely not portable. A type annotation is necessary.

70   render() {
     ~~~~~~


Found 5 errors.
```
![image](https://user-images.githubusercontent.com/22718886/71955967-c8f60800-3224-11ea-9850-f01bbb8d9a5a.png)

--- a/src/components/Drawer/Drawer.tsx
+++ b/src/components/Drawer/Drawer.tsx
@@ -94,7 +94,7 @@ class Drawer extends Component<DrawerProps, DrawerStateInterface> {
     });
   };
 
-  render() {
+  render(): React.ReactPortal | null {
     if (!canUseDOM()) {
       return null;
     }

--- a/src/components/Popup/Popup.tsx
+++ b/src/components/Popup/Popup.tsx
@@ -141,7 +141,7 @@ class Popup extends React.Component<any, any> {
     })
   }
 
-  createPopElement = () => {
+  createPopElement = (): React.ReactPortal => {
     const { className, outStyle, title } = this.props
     const popProperty = {
       key: 'popElement',

--- a/src/components/Portal/Portal.tsx
+++ b/src/components/Portal/Portal.tsx
@@ -94,7 +94,7 @@ class Portal extends PureComponent<ProtalProps, any> {
     return domNode || document.body;
   };
 
-  render() {
+  render(): React.ReactPortal | null {
     // 在服务端不渲染
     if (!canUseDOM()) {
       return null;

--- a/src/components/PurePortal/PurePortal.tsx
+++ b/src/components/PurePortal/PurePortal.tsx
@@ -1,6 +1,6 @@
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-import * as PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
 
 export interface PurePortalProps {
   /**
@@ -67,7 +67,7 @@ class PurePortal extends React.Component<PurePortalProps> {
     this.forceUpdate();
   };
 
-  render() {
+  render(): React.ReactPortal | null {
     if (!this.container) {
       return null;
     }

